serverless/interacting-serverless-apps.adoc

```diff
@@ -6,7 +6,11 @@ include::modules/serverless-document-attributes.adoc[]
 
 toc::[]
 
-This section provides information about various tasks that can be performed once you have successfully created a serverless application.
+This section provides information about various tasks that can be performed once you have successfully created a serverless application using Knative Services.
 
 include::modules/verifying-serverless-app-deployment.adoc[leveloffset=+1]
 include::modules/interacting-serverless-apps-http2-gRPC.adoc[leveloffset=+1]
+include::modules/serverless-interacting-apps-HTTPS.adoc[leveloffset=+1]
+
+== Replacing the default ingress certificate
+Knative Serving uses edge TLS termination of {product-title} Routes. If you want to replace the default, self-signed ingress certificate, follow the documentation on xref:../authentication/certificates/replacing-default-ingress-certificate.adoc#replacing-default-ingress[Replacing the default ingress certificate].
```

Comment:
Since you're putting the link text in a running sentence like this (and not saying "See <section-title>"), I would lowercase "Replacing".